pkg/sql/pgwire/testdata/auth/empty_hba

```diff
@@ -97,3 +97,29 @@ DROP USER testuser_nocert
 ok
 
 subtest end normaluser_nocert
+
+subtest unicode_user_and_username_normalization
+
+sql
+CREATE USER ὀδυσσεύς WITH PASSWORD '蟑♫螂';
+----
+ok
+
+# Verify auth fails without password.
+#
+# Also verify that the username gets normalized: the username
```

Comment:
Do we also need to test here that the non-normalized username (i.e., the one provide initially) also works?

Reply:
Good idea. Done.